src/ext/ddtrace.c

```diff
@@ -226,6 +233,26 @@ static PHP_FUNCTION(dd_trace_forward_call) {
 #endif
 }
 
+static PHP_FUNCTION(dd_trace_cfg) {
+    PHP5_UNUSED(return_value_used, this_ptr, return_value_ptr, ht TSRMLS_CC);
+    PHP7_UNUSED(execute_data);
+    zval *env_name = NULL;
+
+    if (zend_parse_parameters(ZEND_NUM_ARGS() TSRMLS_CC, "z", &env_name) != SUCCESS) {
+        if (DDTRACE_G(strict_mode)) {
+            zend_throw_exception_ex(spl_ce_InvalidArgumentException, 0 TSRMLS_CC,
+                                    "unexpected parameter. the function name must be provided");
```

Comment:
Just a small tweak :)

```suggestion
                                    "unexpected parameter. the environment variable name must be provided");
```

Reply:
Too much copy/paster from me 😄 Thanks for spotting that